packages/amplify-graphql-relational-transformer/src/utils.ts

```diff
@@ -193,6 +193,27 @@ export const ensureReferencesArray = (
   }
 };
 
+export const ensureReferencesBidirectionality = (
```

Comment:
This is a shell of an implementation. Tracking pending work in feature branch PR description as:
> Add validation for relational bi-directionality of DDB references